eng/build.sh

```diff
@@ -136,6 +136,21 @@ showSubsetHelp()
   "$scriptroot/common/build.sh" "-restore" "-build" "/p:Subset=help" "/clp:nosummary"
 }
 
+assertInstalledDependency()
+{
+  dependency="$(echo "$1" | awk '{print tolower($0)}')"
+  location="$(command -v $dependency)"
+
+  if [ -z "$location" ]; then
+    echo "$dependency is required to build this repo. Make sure to install it and try again."
+    echo "For a full list of requirements, see https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/linux-requirements.md"
```

Comment:
The message refers to linux-requirements.md, but the current OS can be OSX.

Reply:
Nice catch. I will add a check and point to OSX when it's such case.